examples/layer-browser/src/data-samples.js

```diff
@@ -1,23 +1,23 @@
 import {Vector3} from '@math.gl/core';
 
 /* load data samples for display */
-import allPoints from '../data/sf.bike.parking.json';
+import allPoints from '../data/sf.bike.parking.json' assert { type: 'json' };
 import {pointGrid} from './utils';
 import {pointsToWorldGrid} from './utils/grid-aggregator';
-import {default as meterTrajectorySmall} from '../data/meter-trajectory-small.json';
-
-import {default as choropleths} from '../data/sf.zip.geo.json';
-export {default as geojson} from '../data/sample.geo.json';
-export {default as geojsonHole} from '../data/hole.geo.json';
-export {default as geojsonLarge} from '../data/canada.geo.json';
-export {default as hexagons} from '../data/hexagons.json';
-export {default as routes} from '../data/sfmta.routes.json';
-export {default as trips} from '../data/trips.json';
-export {default as SFTrips} from '../data/sf.trips';
-export {default as iconAtlas} from '../data/icon-atlas.json';
-export {default as quadkeys} from '../data/sf.quadkeys.json';
-export {default as s2cells} from '../data/sf.s2cells.json';
-export {default as geohashes} from '../data/sf.geohashes.json';
+import {default as meterTrajectorySmall} from '../data/meter-trajectory-small.json' assert { type: 'json' };
+
+import {default as choropleths} from '../data/sf.zip.geo.json' assert { type: 'json' };
```

Comment:
It seems that `assert` is being replaced by `with`: https://stackoverflow.com/questions/70106880/err-import-assertion-type-missing-for-import-of-json-file

Is it possible to use that instead or are you forced to use `assert` because we're targeting older versions of node?

Reply:
Thanks, I am using Node 16 locally. I should upgrade.

Reply:
`node:internal/modules/esm` does not understand `with`. I am keeping `assert` but making sure all tests are run in Node >=18.